circle.yml

```diff
@@ -0,0 +1,17 @@
+machine:
+  python:
+    version: 2.7.9
+  environment:
+    SKIP_CLEANUP: true
+    # comma separated list
+    CIRCLECI_FLAVORS: docker
+
+dependencies:
+  pre:
+    - pip install --use-wheel pyro4==4.35
```

Comment:
this was somehow needed because I didn't override this step, it's no longer the case now that I do.